common.sh

```diff
@@ -349,10 +349,38 @@ create_instance()
     done
 }
 
-# Holds testing every 15 seconds for 40 attempts until the instance status check is ok
+test_instance_status_once()
+{
+    status=$(aws ec2 describe-instance-status  --instance-ids $1 --query InstanceStatuses[*].InstanceStatus.Status --output text)
```

Comment:
If you are planning to create a list, we will need extra round brackets
status=($(aws ec2 describe-instance-status --instance-ids $1 --query InstanceStatuses[*].InstanceStatus.Status --output text))

Reply:
I thought this function is passed one instance-id now. But I am OK with making it flexible.

Reply:
I saw there is a for loop that iterates over status, but I guess we are expecting the status to be string. What would describe-instance-status return if the instance status is not ok. We are checking for ok in the string which would work for both "ok" and "not ok"

Reply:
For the first question, my test show that bash can loop items in a string variable:

```
38f9d35c04a4:memstat_kernel_1.12.0_round2 wzam$ test="ok ok"
38f9d35c04a4:memstat_kernel_1.12.0_round2 wzam$ for i in $test; do echo $i; done
ok
ok
```

Reply:
> We are checking for ok in the string which would work for both "ok" and "not ok"
The output of `describe-instance-status` can also be 'initializing'. If that happens, this function still return "not_ok"

Reply:
Got it, makes sense